stories/SkipLink.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import { ComponentStory, ComponentMeta } from "@storybook/react"
+import { SkipLink } from "../components/SkipLink/SkipLink"
+
+export default {
+	title: "Example/Header",
```

Comment:
The name of the story should be "Example/SkipLink"

Reply:
done